README.md

```diff
@@ -1,5 +1,5 @@
-1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_parse_list_DOM/)
+1. Replace  https://github.com/sinner1993/js_task_parse_list_DOM with your Github username in the link
+    - [DEMO LINK](https://sinner1993.github.io/js_task_parse_list_DOM/)
```

Comment:
Add demo link to the PR description 
<img width="966" alt="image" src="https://github.com/user-attachments/assets/d34dc969-e491-410e-b973-f12c4e6703d4">